scripts/build-worker.sh

```diff
@@ -5,6 +5,16 @@ if [[ -n "${DEBUG-}" ]]; then
   set -x
 fi
 
+# Workaround until the artifact proxying service is set up properly.
```

Comment:
Link to ticket? :)

Reply:
Will have a dig around and see if I can uncover it 😉